mne/beamformer/_dics.py

```diff
@@ -142,7 +703,7 @@ def dics(evoked, forward, noise_csd, data_csd, reg=0.05, label=None,
     real_filter : bool
         If True, take only the real part of the cross-spectral-density matrices
         to compute real filters as in [2]_. Default is False.
-    verbose : bool, str, int, or None
+    verbose : bool | str | int | None
```

Comment:
There are 321 instances of `verbose : bool, str, int, or None` in mne-python and a single one using [`verbose : bool | str | int | or None (default None)`](https://github.com/mne-tools/mne-python/blob/e138e9779dc5f003670effe30a62c3abd6a9e376/mne/cov.py#L651). I would keep the most common, (and change the outlier in a differnt PR)